openrtb_ext/request.go

```diff
@@ -201,7 +202,15 @@ type ExtIncludeBrandCategory struct {
 	TranslateCategories *bool  `json:"translatecategories,omitempty"`
 }
 
+// MediaTypePriceGranularity defines values for bidrequest.ext.prebid.targeting.mediatypepricegranularity
```

Comment:
Consider adding another line in this description specifying what these values mean. Something like:
```
These values specify price granularity configuration at the bid type level.
```